src/streaming-services/common/AutoSync.ts

```diff
@@ -59,7 +59,7 @@ class _AutoSync {
 					...BrowserStorage.options.streamingServices,
 					[serviceId]: {
 						...BrowserStorage.options.streamingServices[serviceId],
-						lastSync: items[0]?.watchedAt?.unix() ?? now,
+						lastSync: now,
```

Comment:
If we save the item watched date here, it could keep triggering the auto sync check every hour, so it's better to save the current date.